From 3003e3ae8ef8100e5d1c3f297dac03750619064e Mon Sep 17 00:00:00 2001
From: Oleksandr Natalenko <oleksandr@redhat.com>
Date: Wed, 28 Aug 2019 07:59:48 +0200
Subject: [PATCH 3/6] Revert "bfq: Fix the missing barrier in
 __bfq_entity_update_weight_prio"

This reverts commit 020caaa615fd8846c1c828bc20f569410ffa7695.
---
 block/bfq-wf2q.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/block/bfq-wf2q.c b/block/bfq-wf2q.c
index 05f0bf4a1144..c9ba225081ce 100644
--- a/block/bfq-wf2q.c
+++ b/block/bfq-wf2q.c
@@ -744,8 +744,6 @@ __bfq_entity_update_weight_prio(struct bfq_service_tree *old_st,
 		}
 #endif
 
-		/* Matches the smp_wmb() in bfq_group_set_weight. */
-		smp_rmb();
 		old_st->wsum -= entity->weight;
 
 		if (entity->new_weight != entity->orig_weight) {
-- 
2.23.0

